src/node_worker.cc

```diff
@@ -380,6 +368,19 @@ void Worker::Run() {
   Debug(this, "Worker %llu thread stops", thread_id_);
 }
 
+void Worker::CreateMessagePort(Environment* env) {
```

Comment:
I’d rename this to make it clear that this refers to the child thread’s main message port, and doesn’t create a generic MessagePort instance

Reply:
Maybe `InitializeMainMessagePort`? Or `CreateMainMessagePort`?

Reply:
`CreateMainMessagePort` or `CreateEnvMessagePort` works fine for me :)